catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -296,6 +173,193 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
     //
     // Private helper methods
     //
+    private DeleteResponse processPostIngestPlugins(DeleteResponse deleteResponse) {
+        for (final PostIngestPlugin plugin : frameworkProperties.getPostIngest()) {
+            try {
+                deleteResponse = plugin.process(deleteResponse);
+            } catch (PluginExecutionException e) {
+                LOGGER.info("Plugin exception", e);
+            }
+        }
+        return deleteResponse;
+    }
+
+    private DeleteResponse processPostDeleteAccessPlugins(DeleteResponse deleteResponse)
+            throws StopProcessingException {
+        for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
+            deleteResponse = plugin.processPostDelete(deleteResponse);
+        }
+        return deleteResponse;
+    }
+
+    private DeleteRequest populateDeleteRequestPolicyMap(DeleteRequest deleteRequest,
```

Comment:
Looks like `deleteResponse` isn't used here.

Also, these methods look so much like each other in all the different `*Operations` classes, we really need to try and reduce that duplication in the follow up PRs.